### DISCUSSION TITLE
General Questions regarding GSoC 2022

### DISCUSSION THREAD

**Discussion by nyrareddy** [koordinates/kart#572] [Google Soc / SoD]
Hello, my name is Nyra. I am planning to apply for GSoC 2022 and would like to start a discussion for any general GSoC related queries. Although the ideas page is quite thorough, I believe people will still have some queries.

Here are few of my questions:
- Is the order in which ideas are listed ordered by highest priority first? If not, are there any high priority projects ?
- How many contributors will be selected for this summer?

It would be great if somebody could help me out on these.

**Comment by rcoup** [comment]
Hi Nyra,

Thanks for getting in touch

> Is the order in which ideas are listed ordered by highest priority first? If not, are there any high priority projects ?

No, there's no priority on our different ideas.

> How many contributors will be selected for this summer?

We don't know, it's up to Google, but as a new project to GSoC we'd be unlikely to get more than 2-3 slots. The Kart mentors rank the different proposals we receive, and we request slots for each quality proposal we'd be happy (and have the capacity) to work on. That happens across all the GSoC projects, and then Google allocates how many slots each project gets and the top-ranked proposals are locked in. Google has some description of this process [here](https://google.github.io/gsocguides/mentor/selecting-a-student#googles-selection-process) and [here](https://google.github.io/gsocguides/mentor/selecting-students-and-mentors) — and there's likely something to do with a mix of medium & large projects in the mix too.

**Comment by ASCE-D** [comment]
Hello Sir ,I'm new to open source and I want to start contributing, Can any mentor guide me on how do I start 
I know python and I can say I'm reaching an intermediate level in it

**Reply by rcoup** [reply]
Hi @ASCE-D

You will need to select or propose an idea, show us you really understand it, and figure out what work it will take to solve. Start a new topic in the discussions and the Kart mentors will chime in to help answer your questions.

**Reply by ASCE-D** [reply]
okay I'll try

**Reply by ASCE-D** [reply]
Hey, can you assign me a good first issue, like I'm totally new and I tried exploring but I don't how to start,,, if you will assign me a small issue then I can try to understand that part so that I can begin with my 1st contribution in open source 
Thank you

**Comment by omsalunke19** [comment]
Hey @rcoup, my name is Om. I am currently studying my Bachelors in Computer Science and Engineering (Specialization in Data Science) (Hons. Blockchain). I was looking through the various organizations and projects under GSOC 2022 aligning with my area of interest and the technologies known to me. GIS is a field I've known since I was in Grade 10 as my father is a professional in that domain. I've worked on ArcGIS Pro and also have a strong hand in Python, with Data Science being in my curriculum.  

I hope to work with you so that I can gain knowledge and experience in GIS, and Geospatial Data Science and upskill myself to a level that would better my chances of landing myself at a better company.

For now, I would like you to mention what all technologies other than ArcGIS would you like me to begin with? 

Thank you for giving me your time.
